### DISCUSSION TITLE
PatchTool 6.4

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#334] [Announcements]
兼容6.4、新增部分翻譯、修正錯誤。

漢化器exe本身有更新。

檔案：

- FFXIVChnTextPatch-20230523.rar：本體。一般而言，下載這個即可。

選用：

- [font-custom-4k-smaller-20230131.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.0.1/font-custom-4k-smaller-20230131.rar)：高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)透過重複使用字體字型縮小檔案體積，使用Noto Sans。
- [font-custom-20230111.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.72.2/font-custom-20230111.rar)：舊高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)所有大小字體皆分開繪製，使用Noto Sans和Noto Serif。
- rawexd-zhcn-20230523.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- FFXIVChnTextPatch.jar：如果你有遇到JNI error，請參考 https://github.com/GpointChen/FFXIVChnTextPatch-GP/issues/90 放到與exe同位置使用。
- backup-20230523.rar：乾淨的0X0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。

※若不替換字體會導致遊戲內無法正常顯示部分中文字。另外，目前大廳部分中文字必須使用高畫質字體包才能正常顯示。更多資訊可參閱[wiki頁面](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v3.1.0.0'>PatchTool 6.4</a>.</em>

**Comment by Manbbb** [comment]
感謝大大神速的更新，辛苦了

**Comment by luckycat6868** [comment]
大大，这次的汉化只有翻译了一半，是我的问题吗 :o

**Reply by GpointChen** [reply]
6.4新增和修改的內容可能會沒有翻譯沒錯

**Comment by luckycat6868** [comment]
![Screenshot 2023-05-25 160808](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/134562946/321119a4-f00f-4a0f-a154-1c5512306137)
![Screenshot 2023-05-25 161321](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/134562946/edeb1fde-40c4-4bf3-b2fb-0f38ba1eda69)
其实本人还没有到最新进度，图中是6.0主线，任务文本也都还是英语

**Reply by luckycat6868** [reply]
刚刚重新下载了一次，问题已经解决了，感谢大大
